docs/api-guide/serializers.md

```diff
@@ -774,6 +774,11 @@ Here's an example of how you might choose to implement multiple updates:
 
     class BookSerializer(serializers.Serializer):
         ...
+        id = serializers.IntegerField(
+            read_only=False,
+            required=False
+        )
```

Comment:
Will do. Sorry for letting this linger. Just had a baby boy join our family last thursday and we are getting to know each other. Will try to get to this by the end of the week.